src/mlpack/methods/ann/layer/isrlu.hpp

```diff
@@ -0,0 +1,138 @@
+/**
+ * @file methods/ann/layer/isrlu.hpp
+ * @author Abhinav Anand
+ *
+ * Definition of the ISRLU activation function as described by Jonathan T. Barron.
+ *
+ * For more information, read the following paper.
+ *
+ * @code
+ * @article{
+ *   author  = {Carlile, Brad and Delamarter, Guy and Kinney, Paul and Marti,
+ *              Akiko and Whitney, Brian},
```

Comment:
Unless there's a reason to have the names in those pairings that I'm unaware of, just list them.
```suggestion
 *   author  = {Carlile, Brad, Delamarter, Guy, Kinney, Paul, Marti, Akiko,
 *              Whitney, Brian},
```

Reply:
Whoops, nevermind.